File: Nested For Loop Pattern-3/code2.c

#include <stdio.h>

void main(){

	int row , temp = 0;
	printf("Enter no. of row : ");
	scanf("%d",&row);

	for(int i = 0;i < row ;i++){
		temp =1;

		for(int a =0; a < row ; a++){

			if(a%2 == 0){

				printf("\t%c\t",row - temp + 65 );
				temp ++;
			}else{

				printf("\t%d\t",1 + i);
			}

			
		}
		printf("\n\n");
	}

}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/12/code2.java


import java.util.Scanner;

class PyramidDemo{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.println("Enter rows : ");
		int row = s.nextInt();
		int temp = 1;

		for(int i = 0 ; i < row ; i++){

			for(int k = 0 ; k  < row-i ; k++){

				System.out.print("		");
			}
			for(int  j = 0 ; j < 2*(i)+1; j++){

				System.out.print(temp+"\t\t");
				temp++;


			}

				System.out.print("\n\n");

		}

	}

}
